Src/Fare.IntegrationTests/XegerTests.cs

```diff
@@ -1,20 +1,43 @@
-﻿using System.Collections.Generic;
+﻿using System;
 using System.Linq;
-using System.Text;
 using Xunit;
+using Xunit.Abstractions;
 
 namespace Fare.IntegrationTests
 {
     public sealed class XegerTests
     {
+        private readonly ITestOutputHelper _testOutput;
```

Comment:
FWIW, ITestOutputHelper is one of the reasons I don't like xUnit 2.* and still prefer xUnit 1.9.* if I have to develop something on the desktop CLR. :sweat_smile:

Reply:
Well, it has its pros and cons. Personally, I accept the fact that I have to use them and don't feel too much pain 😜